lib/ansible/modules/cloud/amazon/_ec2_vpc.py

```diff
@@ -57,7 +57,6 @@
       - 'A dictionary array of subnets to add of the form: { cidr: ..., az: ... , resource_tags: ... }. Where az is the desired availability zone of the subnet, but it is not required. Tags (i.e.: resource_tags) is also optional and use dictionary form: { "Environment":"Dev", "Tier":"Web", ...}. All VPC subnets not in this list will be removed as well. As of 1.8, if the subnets parameter is not specified, no existing subnets will be modified.'
     required: false
     default: null
-    resource_tags: See resource_tags for VPC below. The main difference is subnet tags not specified here will be deleted.
```

Comment:
Suspect the information here should be part of the description of subnets. We should really enforce line length, it makes reading PRs very hard otherwise.